tests/Agent/IntegrationTests/SharedApplications/Common/MultiFunctionApplicationHelpers/MultiFunctionApplicationHelpers.csproj

```diff
@@ -157,8 +130,6 @@
   </ItemGroup>
 
   <ItemGroup> <!-- Versions below 4.5 did not support netstandard 2.0 -->
-    <PackageReference Include="NLog" Version="4.5.10" Condition="'$(TargetFramework)' == 'netcoreapp2.1'" />
-    <PackageReference Include="NLog" Version="4.5.9" Condition="'$(TargetFramework)' == 'netcoreapp2.2'" />
```

Comment:
I'd like to maintain test coverage for the min/max version of NLog we were testing here.

Reply:
I traded one of the test case to test the min version of NLog.